.github/workflows/static-root-checks.yml

```diff
@@ -12,6 +12,8 @@ jobs:
   check:
     if: github.repository == 'software-mansion/react-native-gesture-handler'
     runs-on: ubuntu-latest
+    env:
+      RNGH_DIR: 'packages/react-native-gesture-handler'
```

Comment:
I assume this should be also applied to other actions, and will come in the future?

Reply:
I decided that it will be better to fix CIs at the end, since they require example apps to work. This one doesn't need  them so I've changed it.